### PR TITLE
#127 update the hover color of RNSectionTitle Links on Dark Backgrounds

### LINKED ISSUE #127: Links on dark background still have default hover color
### Component in question:
https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/index.html?path=/story/research-now-section-title--research-now-section-title

### Screenshot of component in your project:
https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/index.html?path=/story/research-now-section-title--research-now-section-title

### Description of issue:
On hover, the link should be a darker shade of grey, not the default hover color

### Overrides (if any) you've added to solve the problem:
```
.example-project {}
```

### PR DESCRIPTION
**Pull request by M0nica**
Resolves: https://github.com/NYPL/nypl-design-system/issues/127

## **This PR does the following:**
- Updates the link hover color of `RNSectionTitle` to be a light grey instead of default link color
- Fixes a syntax issue in `Hero-test.tsx` which had an extra `>`

### Front End Review:
- [ ] View [the example in Storybook](http://localhost:9001/?path=/story/research-now-section-title--research-now-section-title)
- [ ] Check against the [Metronome documentation](http://themetronome.co/components/xxx/?fresh=true)

### CODE REVIEW

**Review by hokei — APPROVED** [review]

**Comment by hokei** [comment]
Remember to update the changelog before merging!

File: src/react-components/CHANGELOG.md
Comment (hokei) [inline src/react-components/CHANGELOG.md]: For now, while we still have different packages going, you want the CSS changes to be in the styles package and the react changes to be here.  So the hero bugfix would go here, and this would go in ``src/styles/CHANGELOG`

Comment (M0nica) [reply src/react-components/CHANGELOG.md]: Thanks, I updated to put the notation in the in the `src/styles/CHANGELOG` and will merge once the CI checks complete.

File: src/styles/CHANGELOG.md
Comment (hokei) [inline src/styles/CHANGELOG.md:11]: another useless piece of information for now until when Lerna is removed - the versioning between `react-components` and `styles` is not the same, this should be `0.0.22`. 
(No biggie!)

**Pull request merged**
